api/tests.py

```diff
@@ -1,6 +1,6 @@
 """Tests for the package.  (Not yet in use.)
 """
 
-from django.test import TestCase
+# from django.test import TestCase
```

Comment:
Add a comment to this explaining that it is commented out to pass the linter.  Leaving it un-commented is an unused import which triggers the linter, but I want to keep this around in case I need it later on (and for a testing module, I certainly should need it later on).

Reply:
Resolved by 1f0c591